src/apis/auth/sign-in-google.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import Elysia, { InternalServerError, t } from "elysia";
+
+import { accessJwt } from "@root/plugins/jwt.plugin";
+import { UserRepository } from "@root/repositories/user.repository";
+import { ENDPOINT } from "@root/shared/constant";
+
+type UserData = {
+  sub: string; // google id
+  email: string;
+};
+
+export const signInGoogle = new Elysia({
+  name: "Handler.SignInGoogle"
+})
+  .use(accessJwt)
+  .post(
+    ENDPOINT.AUTH.SIGN_IN_GOOGLE,
+    async ({ body, accessJwt }) => {
+      const resData = await axios.get(
+        `https://www.googleapis.com/oauth2/v3/userinfo?access_token=${body.accessToken}`
+      );
+
+      const userData: UserData = resData.data;
```

Comment:
Let's move this google api calling into other place. src/services/http/get-google-user-info.ts. This file should include  all function, params type and returns type.